src/controllers/controllermanager.cpp

```diff
@@ -392,15 +392,15 @@ void ControllerManager::slotSavePresets(bool onlyActive) {
         if (onlyActive && !pController->isOpen()) {
             continue;
         }
-        QString name = pController->getName();
-        QString filename = firstAvailableFilename(
-                filenames, sanitizeString(name));
+        QString deviceName = sanitizeString(pController->getName());
+        QString filename = firstAvailableFilename(filenames, deviceName);
         QString presetPath = userPresetsPath(m_pConfig) + filename
                 + pController->presetExtension();
         if (!pController->savePreset(presetPath)) {
             qWarning() << "Failed to write preset for device"
-                       << name << "to" << presetPath;
+                       << deviceName << "to" << presetPath;
         }
+        m_pConfig->set(ConfigKey("[ControllerPreset]", deviceName), presetPath);
```

Comment:
Is this a new ConfigKey? If so, I'd like to get away from this awkward "preset" terminology. No other DJ software uses it, everyone else calls them "mappings". I think "preset" came from a mistaken concept that more than a small fraction of users care to edit their controller mappings but actually they just want them to work. I'm not going to insist on renaming all the preset related code in this PR, but let's not get stuck with this ConfigKey for backward compatibility.

Reply:
No, it exists since at least 1.12: https://github.com/mixxxdj/mixxx/commit/e24a59a25ed2dd479dbcbf8bbc7eacf1eb89ac13#diff-2394e2fc81b545b16e24ea386931aaefR310-R317

Reply:
We could rename, but then users need to setup their controllers again.

Reply:
Okay, let's leave it then. Thanks for checking.